rewrite-hcl/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
 
     compileOnly(project(":rewrite-test"))
 
-    implementation("org.antlr:antlr4:4.9.+")
+    implementation("org.antlr:antlr4:4.11.1")
```

Comment:
When I changed to `4.11.+` it started using 4.11.2-SNAPSHOT. I don't know enough Gradle yet to figure this out, which is why I changed it to `4.11.1`.